server/package.json

```diff
@@ -13,6 +13,7 @@
     "migrate": "node-pg-migrate",
     "feed-participants": "node scripts/feed-participants.js",
     "feed-sites": "node scripts/feed-sites.js",
+    "feed-data": "node scripts/feed-data.js",
```

Comment:
Is this generic enough to be called `feed-data`? It looks like `post_secondary_institutions` is a hard value in the script